packages/maskbook/src/social-network-adaptor/twitter.com/utils/user.ts

```diff
@@ -38,6 +38,18 @@ export const getTwitterId = () => {
 
 export const getBioDescription = () => {
     const node = bioDescriptionSelector().evaluate()
+    if (node?.children) {
```

Comment:
Please try to reuse this function:

https://github.com/DimensionDev/Maskbook/blob/ad14cf2692615fa04e31bf48cf7cb2621ebf2e22/packages/maskbook/src/social-network-adaptor/twitter.com/utils/fetch.ts#L43-L43

Reply:
Good suggestion. I'll try later.